backend/main/server/app.go

```diff
@@ -590,8 +590,22 @@ func (a *App) createVoteForProposal(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	s := strategyMap[*p.Strategy]
+	if s == nil {
+		respondWithError(w, http.StatusInternalServerError, "Strategy not found")
+		return
+	}
+
+	// create the voteWithBalance struct
+	vb := models.VoteWithBalance{
```

Comment:
I think the balance data is missing from this model, so `s.GetWeightForBalance` won't work correctly